src/components/card/card--test.njk

```diff
@@ -0,0 +1,193 @@
+          <h3 class="usa-card__heading">A quality even meaner than outright ugliness or disorder</h3>
+        </div>
+        <div class="usa-card__media usa-card__media--exdent">
+          <div class="usa-card__img">
+            <img
+              src="https://source.unsplash.com/featured/800x450/daily?patterns"
+              alt="">
+          </div>
+        </div>
+        <div class="usa-card__body">
+          <p>There is a quality even meaner than outright ugliness or disorder, and this meaner quality is the dishonest mask of pretended order, achieved by ignoring or suppressing the real order that is struggling to exist and to be served.</p>
+        </div>
+        <div class="usa-card__footer">
+          <a href="javascript:void(0);" class="usa-button">Pursue a better life</a>
+        </div>
+      </div>
+    </li>
+    <li class="usa-card usa-card--media-right usa-card--flag tablet:grid-col-12">
+      <div class="usa-card__container border-primary-vivid">
+        <div class="usa-card__header">
+          <h3 class="usa-card__heading">Pretend order</h3>
+        </div>
+        <div class="usa-card__media flex-align-center">
+          <div class="usa-card__img circle-card margin-x-auto">
+            <img
+              src="https://source.unsplash.com/featured/450x600/daily?human"
+              alt="">
+          </div>
+        </div>
+        <div class="usa-card__body">
+          <p>There is a quality even meaner than outright ugliness or disorder, and this meaner quality is the dishonest mask of pretended order, achieved by ignoring or suppressing the real order that is struggling to exist and to be served.</p>
+        </div>
+        <div class="usa-card__footer">
+          <a href="javascript:void(0);" class="usa-button usa-button--outline">Pursue a better life</a>
+        </div>
+      </div>
+    </li>
+    <li class="usa-card usa-card--header-first tablet:grid-col-6 desktop:grid-col-4">
+      <div class="usa-card__container">
+        <div class="usa-card__header">
+          <h3 class="usa-card__heading">The death and life of great American cities</h3>
+        </div>
+        <div class="usa-card__media usa-card__media--exdent usa-card__media--fix-aspect add-aspect-1x1">
+          <div class="usa-card__img">
+            <img
+              src="https://source.unsplash.com/featured/450x600/daily?people"
+              alt="">
+          </div>
+        </div>
+        <div class="usa-card__body">
+          <p>There is a quality even meaner than outright ugliness or disorder, and this meaner quality is the dishonest mask of pretended order, achieved by ignoring or suppressing the real order that is struggling to exist and to be served.</p>
+        </div>
+        <div class="usa-card__footer">
+          <a href="javascript:void(0);" class="usa-button">Pursue a better life</a>
+        </div>
+      </div>
+    </li>
+    <li class="usa-card tablet:grid-col-6 desktop:grid-col-fill">
```

Comment:
Media is getting stretched in IE11.

![image](https://user-images.githubusercontent.com/3385219/80503960-31820580-8938-11ea-9c1f-14167d6e60e2.png)